precompiles/erc20/events_test.go

```diff
@@ -0,0 +1,73 @@
+package erc20_test
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	cmn "github.com/evmos/evmos/v15/precompiles/common"
+	erc20precompile "github.com/evmos/evmos/v15/precompiles/erc20"
+	utiltx "github.com/evmos/evmos/v15/testutil/tx"
+)
+
+//nolint:dupl // this is not a duplicate of the approval events test
+func (s *PrecompileTestSuite) TestEmitTransferEvent() {
```

Comment:
can we structure this as a table test instead? See @Vvaradinov's test on outpost for reference

Reply:
there's no possibility to let this test fail so the "table" only consists of one entry 😅 that's unnecessary boilerplate IMO and I would rather suggest to also remove that from the other tests